app/lib/forms/question_types/base.rb

```diff
@@ -1,12 +1,13 @@
 module Forms
   module QuestionTypes
     class Base
-      attr_reader :name, :options
+      attr_reader :name, :options, :locale_name, :question_data
```

Comment:
`question_data` is not assignable via initializer. Is that on purpose?

Reply:
That is a mistake and should've been removed when I reverted a possible avenue for implementing something. Removing this now.